packages/client/hooks/usePortal.tsx

```diff
@@ -38,7 +38,7 @@ export interface UsePortalOptions {
   onClose?: () => void
   id?: PortalId
   // if you nest portals, this should be the id of the parent portal
-  parentId?: PortalId
+  parentId?: PortalId | string
```

Comment:
-1 the parent portal is enumerable, if you need to change it to `string` it becomes generic, need to rethink about it

Reply:
Never mind, I was able to get a workaround, pushing soon.